README.md

```diff
@@ -1,4 +1,4 @@
-# WHAT backend
+<h1 align="center">WHAT backend</h1>
```

Comment:
I think we would prefer this header to be written in markdown format and to have a default align value.
But thanks for the suggestion!